test/expected/delete.out

```diff
@@ -59,3 +59,126 @@ SELECT * FROM "two_Partitions" ORDER BY "timeCustom", device_id;
  1257897600000000000 | dev1      |      4.5 |        5 |          | f
 (6 rows)
 
+-- Make sure DELETE isn't optimized if it includes Append plans
+-- Need to turn of nestloop to make append appear the same on PG96 and PG10
+set enable_nestloop = 'off';
```

Comment:
I don't see why this plan would be optimized by ConstraintAwareAppend otherwise. Isn't it better to use time and now() or something. Can you explain?

Reply:
Yes, you are right. I apparently changed my tests in a dumb way. Will fix.